src/modules/cmdpal/exts/Microsoft.CmdPal.Ext.WinGet/Pages/InstallPackageListItem.cs

```diff
@@ -49,16 +49,24 @@ public InstallPackageListItem(CatalogPackage package)
 
     private Details? BuildDetails(PackageVersionInfo? version)
     {
-        CatalogPackageMetadata? metadata = version?.GetCatalogPackageMetadata();
+        var metadata = version?.GetCatalogPackageMetadata();
         if (metadata != null)
         {
+            if (metadata.Tags.Where(t => t.Equals(WinGetExtensionPage.ExtensionsTag, StringComparison.OrdinalIgnoreCase)).Any())
```

Comment:
okay so... we're checking whether we're installing a package for the extensions page _based on whether it's got the right tag_? I do NOT love this. File a bug to remove it.

Reply:
I did it this way so that finding an extension on the main winget page, and trying to install it from there would still Just Work (rather than make the extensions page Special)